presto-main/src/test/java/com/facebook/presto/security/TestFileBasedSystemAccessControl.java

```diff
@@ -194,16 +199,67 @@ public void testEverythingImplemented()
         assertAllMethodsOverridden(SystemAccessControl.class, FileBasedSystemAccessControl.class);
     }
 
+    @Test
+    public void testRefreshing()
+            throws Exception
+    {
+        TransactionManager transactionManager = createTestTransactionManager();
+        AccessControlManager accessControlManager = new AccessControlManager(transactionManager);
+        File configFile = newTemporaryFile();
+        configFile.deleteOnExit();
+        copy(new File(getResourcePath("catalog.json")), configFile);
+
+        accessControlManager.setSystemAccessControl(FileBasedSystemAccessControl.NAME, ImmutableMap.of(
+                SECURITY_CONFIG_FILE, configFile.getAbsolutePath(),
+                SECURITY_REFRESH_PERIOD, "1s"));
```

Comment:
why not 10ms ?

now this test takes 4s, could be faster.

Reply:
This test is time-based and it is very easy to make it flaky, so I prefer a longer period.